apps/web/src/studio/components/workflows/node-view/WorkflowFloatingMenu.tsx

```diff
@@ -14,63 +14,74 @@ import {
 } from '@novu/novui/icons';
 import { VStack } from '@novu/novui/jsx';
 import { vstack } from '@novu/novui/patterns';
-import { FC, PropsWithChildren } from 'react';
+import { FC, PropsWithChildren, useState } from 'react';
+import { DocsModal } from '../../../../components/docs/DocsModal';
 
 type IWorkflowFloatingMenuProps = CoreProps;
 
 export const WorkflowFloatingMenu: FC<IWorkflowFloatingMenuProps> = ({ className }) => {
-  // TODO: this is just a temporary hack. Behavior should be passed through props
-  const handleClick = (stepType: string) => () => {
-    alert('This should open docs for ' + stepType);
+  const [docsOpen, setDocsOpen] = useState<boolean>(false);
+  const [path, setPath] = useState<string>('');
+
+  const toggle = () => {
+    setDocsOpen((prevOpen) => !prevOpen);
+  };
+
+  const handleClick = (pathToSet: string) => () => {
+    setPath('echo/steps/' + pathToSet);
```

Comment:
To make it easy for use to switch when we have changed echo in our docs path

Reply:
What does this `setPath` do? Is it a navigation?

Reply:
It sets the path of what docs the modal should render.